src/components/BlogCreation.vue

```diff
@@ -4,7 +4,7 @@
         <div>
             <input class="titleStyle" type="text" placeholder="Enter a title!" v-model="NewPost.title" />
             <textarea class="textStyle"  placeholder="Enter some details!" v-model="NewPost.info" rows="5" />
-            <input class="buttonStyle" type="button" value="Submit!" />
+            <input class="buttonStyle" type="button" value="Submit!" @click="$emit('upload-post', NewPost.id)"/>
```

Comment:
@ricomadeit If it is sending the post's data to the parent for uploading or displaying, I'd send through the entire NewPost like this `@click="$emit('upload-post', NewPost)"`